App/server/models/garden.py

```diff
@@ -21,14 +22,14 @@ class Config:
 
         @root_validator
         def number_validator(cls, values):
-            values["updated_at"] = datetime.utcnow()
+            values["updated_at"] = datetime.now(pytz.timezone("US/Eastern"))
             return values
 
 
 class GardenUpdate(BaseModel):
     name: Optional[str]
     location: Optional[str]
-    config_id: Optional[str]
+    config_id: str = None
```

Comment:
why is config_id = None here? I think it's possible this might reset the config_id to None every time garden update is called

Reply:
Yeah I think not having it as None makes more sense. The change was made initially because the garden routes were failing if the config_id was not part of the query even though it was supposedly optional.